### PR TITLE
BRAIN-39 - Replaced webpack with vite

### PR DESCRIPTION
**Pull request by cyl3x**
- Replaced webpack with vite and slim down deps
    - dev bundle from 24MB to 16MB
    - prod bundle from 9.5MB to 8.8MB/6.3MB
- Removed unnecessary eslint bloat
- Added hmr mode though `npm run watch`

### CODE REVIEW

**Comment by lernhart** [comment]
Will this build a single entrypoint .js and .css file ?

**Comment by cyl3x** [comment]
It will chuck the build, producing 3 entry points (total 8.8MB):
```
4,6M    public/build/admin.css (4MB fonts)
640K    public/build/admin.js
88K     public/build/polyfills.js
3.4M    public/build/assets (async loaded files)
```

We could bundle it into single files to allow better client side caching (total 6.3MB):
```
4.6M    public/build/admin.css
1.5M    public/build/admin.js (size due to included meteor-icons)
88K     public/build/polyfills.js
```

**Review by Bird87ZA — APPROVED** [review]
I don't know much about vite, but there's no glaring issue popping out to me

File: templates/admin-sdk.html.twig [hunk 1]
```diff
@@ -6,12 +6,17 @@
...
-    {{ encore_entry_link_tags('admin') }}
-    {{ encore_entry_script_tags('admin') }}
+    {% if hmr %}
+    <script type="module" rel="script" src="/_vite_/@vite/client"></script>
+    <script type="module" rel="script" src="/_vite_/assets/admin.ts"></script>
+    {% else %}
+    <link rel="stylesheet" href="/build/admin.css">
+
+    <script type="module" src="/build/admin.js"></script>
+    <script type="module" src="/build/polyfills.js"></script>
```
Comment (mstegmeyer) [inline templates/admin-sdk.html.twig:16]: could we make these paths subdirectory-able?

Comment (cyl3x) [reply templates/admin-sdk.html.twig:16]: We could, at least in my development environment this works - just for the record, the encore build before also only outputs absolute paths

**Pull request merged**